### ISSUE TITLE
Swift compiler warning: Integer overflows when converted from 'Builtin.Int32' to 'Builtin.Int8'

### ISSUE THREAD

**Issue by master-nevi** [CocoaLumberjack/CocoaLumberjack#471]
I'm importing CocoaLumberjack using CocoaPods 0.36.0.rc.1 with Xcode 6.1.1. No build errors however I do get that swift compiler warning.
```
warning: Integer overflows when converted from 'Builtin.Int32' to 'Builtin.Int8'
```

The error comes from the current version of CocoaLumberjack.swift (imported directly into my swift project) when using StaticString.stringValue. I've been able to prevent the warning by using Swift string interpolation of StaticString instead of StaticString.stringValue as shown here: https://github.com/master-nevi/CocoaLumberjack/commit/2b41f894a47c6843103edc1184441aaaee425eed . I got the idea from this post on stack overflow: http://stackoverflow.com/questions/25867546/swift-warning-integer-overflows-when-converted-from-builtin-int32-to-builti.

If you think this is an appropriate fix I can create a pull request, however it seems odd that this warning would occur.

**Comment by MaddTheSane** [comment]
That seems more like a compiler bug. At least now I know where it comes from.

**Comment by rivera-ernesto** [comment]
I agree with both. A pull request for temporary fix, and a short comment about it in code, would be great.

**Comment by master-nevi** [comment]
Done and done: https://github.com/CocoaLumberjack/CocoaLumberjack/pull/472

**Comment by bpoplauschi** [comment]
Fixed by #472
